extension/copy-file.js

```diff
@@ -18,3 +18,32 @@ window.addFileCopyButton = () => {
 		utils.copyToClipboard(fileContents);
 	});
 };
+window.addFilePathCopyButton = () => {
```

Comment:
Locally this whole section throws an error in `npm test` because of `xo` errors.